portal-web/test/functional/com/liferay/portalweb/paths/mypaths/MyPath.path

```diff
@@ -0,0 +1,31 @@
+<html>
+<head>
+<title>MyPath</title>
+</head>
+
+<body>
+<table border="1" cellpadding="1" cellspacing="1">
+<thead>
+<tr><td rowspan="1" colspan="3">MyPath</td></tr>
+</thead>
+
+<tbody>
+<tr>
+	<td>APPLICATIONS_MENU</td>
+	<td>//div[contains(@class,'control-menu')]//button[@data-qa-id='applicationsMenu']</td>
+	<td></td>
+</tr>
+<tr>
+	<td>CONTROL_PANEL</td>
+	<td>//li[@class='nav-item']/button[child::span[text()='Control Panel']]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>PANEL_CONFIGURATION_TEXT</td>
+	<td>//div[@class='col-md']//h2[@id='nav_control_panel.configuration']</td>
+	<td></td>
```

Comment:
you could also just use `//h2[@id='nav_control_panel.configuration']` but you don't have to change that, your xpath is also correct this way, thanks!

Reply:
Okay, noted! :+1: